pkg/webservice/handlers.go

```diff
@@ -1172,6 +1176,14 @@ func getStream(w http.ResponseWriter, r *http.Request) {
 	stream := eventSystem.CreateEventStream(r.Host, count)
 	defer eventSystem.RemoveStream(stream)
 
+	if err := enc.Encode(YunikornID{
+		InstanceUUID: schedulerContext.GetUUID(),
+	}); err != nil {
+		buildJSONErrorResponse(w, err.Error(), http.StatusInternalServerError)
```

Comment:
The reason of using `500` instead of `200` is that this is the first event at this point?

Reply:
Yes. We could use 500 later on as well, but they wouldn't have any impact. Any write to the response body automatically sets 200 OK, because that must be sent beforehand.